src/pip/_internal/vcs/git.py

```diff
@@ -94,6 +94,7 @@ def get_git_version(self) -> Tuple[int, ...]:
         version = self.run_command(["version"], show_stdout=False, stdout_only=True)
         match = GIT_VERSION_REGEX.match(version)
         if not match:
+            logger.warning("Unable to parse '%s'.", version)
```

Comment:
It may also be worth while to explain where we got that unparsable string, i.e.

```suggestion
            logger.warning("Unable to parse 'git --version' output: %r.", version)
```

Reply:
Do you mean something like the below?
`
# Equivalent of running 'git --version'
# See run_command in parent class
`

Reply:
I don’t quite understand what you meant. What I was trying to express is, this *Unable to parse* message will be seen by the user in the console, and when that happens the user may not fully understand the context of the message (why does the string fail to parse? Where does that string come from?) Therefore, I recommended adding *git version output* to the message, so the user understands the string is the output to `git version` and react accordingly (and yes, `git --version` gives the exact same output, but we are using `git version` right now so that should be used instead).

Reply:
Ah okay. I misunderstood, my apologies.